gunicorn/arbiter.py

```diff
@@ -494,6 +496,11 @@ def manage_workers(self):
 
     def spawn_worker(self):
         self.worker_age += 1
+        worker_id = -1
+        for i in range(self.num_workers):
```

Comment:
The only issue with using worker.age is if a worker is reaped and a new one is spawned in its place, worker.age continues to increment and the instance id will not remain in the anticipated range.